daisy/compute/compute.go

```diff
@@ -1026,3 +1038,19 @@ func (c *client) SetCommonInstanceMetadata(project string, md *compute.Metadata)
 
 	return c.i.globalOperationsWait(project, op.Name)
 }
+
+// GetGuestAttributes gets a Guest Attributes.
+func (c *client) GetGuestAttributes(project, zone, name, queryPath, variableKey string) (*computeBeta.GuestAttributes, error) {
+	call := c.rawBeta.Instances.GetGuestAttributes(project, zone, name)
+	if queryPath != "" {
+		call = call.QueryPath(queryPath)
+	}
+	if variableKey != "" {
```

Comment:
should this be nested in the previous block? Can you have a variableKey but no queryPath?

Reply:
this is just an api wrapper, the api lets you send one or the other or both so this function does too